### PR TITLE
test(asyncapi): align tests with oas 

### PR DESCRIPTION
**Pull request by P0lip**
Extracted from https://github.com/stoplightio/spectral/pull/1470
Related to #1082 
Complementary PR to https://github.com/stoplightio/spectral/pull/1472
It moves us one step ahead towards monorepo.

**Checklist**

- [x] Tests added / updated
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/rulesets/asyncapi/__tests__/asyncapi-operation-description.ts [hunk 1]
```diff
@@ -1,31 +1,30 @@
-import { cloneDeep } from 'lodash';
-
-import { buildTestSpectralWithAsyncApiRule } from '../../../../setupTests';
-import { Rule } from '../../../rule';
 import { Spectral } from '../../../spectral';
+import { loadRules } from './__helpers__/loadRules';
+import { DiagnosticSeverity } from '@stoplight/types/dist';
```
Comment (domagojk) [inline src/rulesets/asyncapi/__tests__/asyncapi-operation-description.ts:3]: why from `types/dist`?

File: src/rulesets/asyncapi/__tests__/__helpers__/loadRules.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { STATIC_ASSETS } from '../../../../assets';
+import { empty } from '../../../../utils';
+import { Spectral } from '../../../../spectral';
+import { isAsyncApiv2 } from '../../../../formats';
+import * as ruleset from '../../index.json';
+
+export async function loadRules(rules: (keyof typeof ruleset['rules'])[]): Promise<Spectral> {
```
Comment (domagojk) [inline src/rulesets/asyncapi/__tests__/__helpers__/loadRules.ts:7]: Same as https://github.com/stoplightio/spectral/pull/1472#discussion_r556464162

File: src/rulesets/asyncapi/__tests__/asyncapi-operation-description.ts (hunk 1 above)
Comment (P0lip) [reply src/rulesets/asyncapi/__tests__/asyncapi-operation-description.ts:3]: WebStorm does it by default at times :laughing: 
Will change

Comment (domagojk) [reply src/rulesets/asyncapi/__tests__/asyncapi-operation-description.ts]: obscure software

**Review by domagojk — APPROVED** [review]

**Pull request merged**